.changeset/calm-horses-feel.md

```diff
@@ -0,0 +1,11 @@
+---
+"undefined": minor
+"@fluidframework/core-interfaces": minor
+"@fluidframework/tree": minor
+---
+---
+"section": other
+---
+
+
+Relocate the events library from SharedTree by moving its types and interfaces to `@fluidframework/core-interfaces` and its implementation to `@fluid-internal/client-utils`.
```

Comment:
Maybe call out if there's any changes to the expected usage (or if the usage stays the same).